util/src/main/java/io/camunda/zeebe/util/sched/ActorMetrics.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.util.sched;
+
+import io.prometheus.client.Counter;
+import io.prometheus.client.Histogram;
+
+public class ActorMetrics {
```

Comment:
```suggestion
final class ActorMetrics {
```

Reply:
:wrench: As a rule of thumb, I would generally keep the lowest visibility required if only to make sure when other classes depend on one (i.e. it must be made public) hopefully we think about it and what it means to couple them.

Reply:
I guess I should just copy old branches :sweat_smile: BTW why we still have no check for class needs to be final ? :thinking: I guess it is not easy to detect ? :thinking: Or what is the reason?